packages/aws-cdk-lib/pipelines/README.md

```diff
@@ -13,7 +13,7 @@ CDK container assets for ECS clusters instead.
 
 Give the CDK Pipelines way of doing things a shot first: you might find it does
 everything you need. If you need more control, or if you need `v2` support from
-`aws-codepipeline`, we recommend you drop down to using the `aws-codepipeline` 
+`aws-codepipeline`, we recommend you drop down to using the `aws-codepipeline`
```

Comment:
this change is unrelated and unnecessary (for commit objective), right?

Reply:
@ChakshuGupta13 
Thank you. Unnecessary spaces were automatically deleted.
I reverted it.​​​​​​​​​​​​​​​​